### ISSUE TITLE
Close repo

### ISSUE THREAD

**Issue by julibert** [micro-ROS/micro-ROS_experiments#1]
This repo seems unmaintained. Could we remove it?

**Comment by ralph-lange** [comment]
Yes, can be removed. The rclcpp fork, which was used in one experiment can be also removed. I already archived micro-ros/rclcpp.
